### DISCUSSION TITLE
[QA] PROFILE

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#269] [Q&A]
PROFILE

**Comment by YuHyun-P** [comment]
![image](https://user-images.githubusercontent.com/96400112/174742521-55864717-f817-4b66-9145-47eb41b8e6d6.png)

자신의 프로필인지 검증하지 않기 때문에 자신의 프로필이라면 비밀번호 변경이 되고 아니면 안 됩니당

**Reply by YuHyun-P** [reply]
게시글 갯수가 부정확하게 뜨는 이유는 버그를 고치기 전에 생성한 글이 있어서 그런 것 같습니다!
![image](https://user-images.githubusercontent.com/96400112/174749424-e7e7a922-2f41-496e-b45c-9d3487fb2699.png)
![image](https://user-images.githubusercontent.com/96400112/174750017-12200290-30f2-4ed8-bb8c-d8e001ffe39c.png)

**Reply by YuHyun-P** [reply]
아직 팔로우 팔로워 기능은 없습니다

**Reply by Hyevvy** [reply]
헙 그러면 이거 다 게시글 지워야할까용 ?

**Comment by Hyevvy** [comment]
마이페이지에서 하트가 클릭이 안 되면 하트를 아예 안 보이게 하는게 낫지 않을까용 ?? 
다른 곳에서는 토글이라서 뭔가 혼동이 될 거 같네용 ! !

**Comment by Hyevvy** [comment]
이걸 어디다가 리포트할 지 몰라서 우선은 상세페이지랑 프로필페이지 두 군데 모두 적습니다 

상세페이지에서 유저 프로필로는 가지는데 이 때 마이페이지로 이동이 안 돼용 

https://user-images.githubusercontent.com/72402747/174765811-0f9b8526-ac90-400a-9424-9b104f4676f0.mov

**Reply by YuHyun-P** [reply]
#305 해결😭

**Comment by Hyevvy** [comment]
더볼 게 없는데 더보기 버튼이 떠버리네용 ㅠㅠ 

https://user-images.githubusercontent.com/72402747/174768066-5f753c64-7ffd-423c-98e7-8c1d6e202708.mov

**Reply by YuHyun-P** [reply]
#305 해결😭
